templates/rpm.erb

```diff
@@ -43,7 +43,7 @@ Epoch: <%= epoch %>
 <% end -%>
 Release: <%= iteration or 1 %><%= "%{?dist}" if attributes[:rpm_dist] %>
 <%# use the first line of the description as the summary -%>
-Summary: <%= description.split("\n").first || "_" %>
```

Comment:
// , In older versions, this works as a way to add a summary by using the part in the --description argument before the newline, right?